Rest/client.py

```diff
@@ -0,0 +1,31 @@
+import requests
+
+class Task_rest():
+
+    def __init__(self, url, parameter_for_query):
+        self.url = url
+        self.parameter_for_query = parameter_for_query
+
+    def get(self):
+        query_get = requests.get(self.url, params=self.parameter_for_query)
+        return query_get
+
+    def delete(self, ):
+        query_delete = requests.delete(self.url, params=self.parameter_for_query)
+        return query_delete
+
+    def post(self):
+        query_post = requests.post(self.url, json=self.parameter_for_query)
+        return query_post
+
+    def put(self):
+        query_put = requests.put(self.url, params=self.parameter_for_query)
+        return query_put
+
+def message_text_from_response(message):
+    message_from_response_split = message.split(':')
```

Comment:
Why we use it method?
you should get .json reason and status code from response